src/Renci.SshNet.Tests/Common/TripleATestBase.cs

```diff
@@ -24,6 +25,10 @@ protected virtual void TearDown()
         protected abstract void Arrange();
 
         protected abstract void Act();
+
+        protected virtual void AfterAct()
```

Comment:
When would or do you use this?

Reply:
I override this method in `HttpConnectorTestBase`, `Socks4ConnectorTestBase`, `Socks5ConnectorTestBase`. Example implementation: 
https://github.com/sshnet/SSH.NET/blob/984c5c360bd115f2a60c249c5221dbf65879b024/src/Renci.SshNet.Tests/Classes/Connection/HttpConnectorTestBase.cs#L29-L33

NET7 is much more efficient so tests were performed first and then events like `Disconnected`, `BytesReceived`. For instance: https://github.com/sshnet/SSH.NET/blob/984c5c360bd115f2a60c249c5221dbf65879b024/src/Renci.SshNet.Tests/Classes/Connection/Socks4ConnectorTest_Connect_ConnectionRejectedByProxy.cs#L33-L48 

The problem occurs especially when running tests from the console.

Reply:
I prefer to remove this **AfterAct** method, and where necessary ntegrate the sleep in the **Act** for those tests that need it. OK for you, or want to discuss this further?

Reply:
It will be ok for me.

Reply:
I've removed the `AfterAct` method and added `Thread.Sleep(200)` for all tests that use `AsyncSocketListener`.